experiments/launch_sixteen_heads.py

```diff
@@ -254,7 +254,7 @@ def replace_masked_hook_points(model):
         name = f"blocks.{layer_i}.hook_mlp_out"
         mlp_nodes = [
             TLACDCInterpNode(name, TorchIndex([None]), incoming_edge_type=EdgeType.PLACEHOLDER),
-            TLACDCInterpNode(f"blocks.{layer_i}.hook_resid_mid", TorchIndex([None]), incoming_edge_type=EdgeType.ADDITION),
+            TLACDCInterpNode(f"blocks.{layer_i}.hook_mlp_in", TorchIndex([None]), incoming_edge_type=EdgeType.ADDITION),
```

Comment:
Did this name change?

Reply:
TL;DR Yes --- this is needed so that we can just import the latest TransformerLens version rather than rely on patched versions

Extra detail: the change is implemented here https://github.com/neelnanda-io/TransformerLens/pull/313 and the linked issue explains the problem